lib/ezutils/classes/ezsendmailtransport.php

```diff
@@ -31,19 +31,15 @@ function sendMail( eZMail $mail )
         if ( !eZMail::validate( $emailSender ) )
             $emailSender = false;
 
-        $isSafeMode = ini_get( 'safe_mode' ) != 0;
-
         $sendmailOptionsArray = $ini->variable( 'MailSettings', 'SendmailOptions' );
         if( is_array($sendmailOptionsArray) )
             $sendmailOptions = implode( ' ', $sendmailOptionsArray );
         elseif( !is_string($sendmailOptionsArray) )
             $sendmailOptions = $sendmailOptionsArray;
-        if ( !$isSafeMode and
-             $emailSender )
+        if ( $emailSender )
             $sendmailOptions .= ' -f'. $emailSender;
 
-        if ( $isSafeMode and
-             $emailSender and
+        if ( $emailSender and
              $mail->sender() == false )
             $mail->setSenderText( $emailSender );
```

Comment:
should this still be here?
(seems emailSender is set above for non safe mode, so seems it's set two times now in different ways)

Reply:
Fixed in f4fb830